src/App.jsx

```diff
@@ -1,12 +1,30 @@
 import { useState } from 'react'
 import './App.css'
+import { Routes, Route, Router } from 'react-router-dom'
+import { Home } from './components/Home'
+import { Explore } from './components/Explore'
+import { Post } from './components/Post'
+import { LogIn } from './components/LogIn'
+import { Account } from './components/Account'
+import { Header } from './components/Header'
 
 function App() {
 
+  const [articles, setArticles] = useState([])
+
   return (
-    <>
-    
-    </>
+  <>
+      <Header></Header>
+    <Routes>
+      <Route path='/' element={<Home />}/>
+      <Route path='/explore' element={<Explore articles={articles} setArticles={setArticles}/>}/>
```

Comment:
At the moment you're passing down articles and setArticles but no using it anywhere else. Remember that state should be as low down as possible but as high up as needed, you could just have your state in the "Explore" component.